app/src/chat_api.py

```diff
@@ -367,6 +367,8 @@ class Citation(BaseModel):
     uri: Optional[str] | None
     headings: Sequence[str]
     citation_text: str
+    chunk_id: str
+    subsection_index: int
```

Comment:
Added so that API client can merge contiguous citations. Hopefully, the client side will ignore extra attributes in the response. TODO: Should notify them in Slack.